nx.json

```diff
@@ -135,14 +135,9 @@
     },
     "remix-ui-tabs": {
       "tags": []
+    },
+    "remix-ui-theme-module": {
+      "tags": []
     }
-  },
-  "targetDependencies": {
```

Comment:
this is need in order for the nx build remix-ide to build deps

Reply:
we don't use with-deps anymore

Reply:
wow...thanks man!